### PR TITLE
docs: Add a decision around repo layout.

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by feanil** [comment]
@Carlos-Muniz finally getting around to writing this down [as you suggested](https://github.com/openedx/docs.openedx.org/pull/36#pullrequestreview-1002839183) :joy:

File: source/documentors/decisions/0003-documentation-layout.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This made it harder for someone to easily jump between work on different
+personas and easily locate documents and orient themselves.
+
+Decision
+********
+
+* The ``source`` folder MUST be the root for all docs.
+
+* For each top-level persona that we have, there MUST be a separate folder whose
+  name closely matches the name of the persona.
```
Comment (sarina) [inline source/documentors/decisions/0003-documentation-layout.rst:23]: what does this mean?

File: source/documentors/decisions/0003-documentation-layout.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+   ├── references
+   │   └── index.rst
+   └── index.rst
+
+Consequences
+************
+
+Having a consistent layout should make it easier for documentors to be able to
+navigate the repository and quickly switch between personas and know where to
+put new documentation and where to find the roots for each sub-area of a persona.
```
Comment (Carlos-Muniz) [inline source/documentors/decisions/0003-documentation-layout.rst:57]: Should there be a new line at the end of this doc?

Comment (feanil) [reply source/documentors/decisions/0003-documentation-layout.rst:66]: There is one, if there wasn't github would indicate that.  It just doesn't print the trailing blank line.

**Pull request merged**